specification/resources/databases/models/database_layout_options.yml

```
@@ -0,0 +1,8 @@
+type: object
+
+properties:
+  layouts:
+    type: array
+    readOnly: true
```

```suggestion
    readOnly: true
    description: Layouts describe....
```

Would be great to have a description here. I'm not sure "layouts" are a well understood concept for users.